tests/test_usage.py

```diff
@@ -336,11 +338,61 @@ def test_hear_works_WHEN_say_works(self):
 			theResult = (int(p.exitcode) <= int(0))
 		except Exception as err:
 			context.debugtestError(err)
-			# raise unittest.SkipTest(fail_fixture)
 			self.fail(fail_fixture)
 			theResult = False
 		self.assertTrue(theResult, fail_fixture)
 
+	@staticmethod
+	def _always_generate_random_port_WHEN_called():
+		"""Outputs a psuedo-random, RFC-6335 compliant, port number."""
+		return _random.randint(49152, 65535)
+
+	def test_hear_works_WHEN_fuzzed_and_say_works(self):
+		"""Tests the basic send and recv test. Skips if fuzzing broke SAY fixture."""
+		theResult = False
+		fail_fixture = str("""SAY --> HEAR == error""")
+		_fixture_port_num = self._always_generate_random_port_WHEN_called()
+		try:
+			self.assertIsNotNone(_fixture_port_num)
+			self.assertEqual(type(_fixture_port_num), type(int(0)))
+			_fixture_SAY_args = [
+				"""--port""", str(_fixture_port_num),
+				"""--mcast-group""", """'224.0.0.1'""",
+				"""--message""", """'test message'"""
+			]
+			_fixture_HEAR_args = [
+				"""--port""", str(_fixture_port_num),
+				"""--join-mcast-groups""", """'224.0.0.1'""",
+				"""--bind-group""", """'224.0.0.1'"""
+			]
+			p = Process(
+				target=multicast.__main__.McastDispatch().doStep,
+				name="HEAR", args=("HEAR", _fixture_HEAR_args,)
+			)
+			p.start()
+			try:
+				self.assertIsNotNone(
+					multicast.__main__.McastDispatch().doStep("SAY", _fixture_SAY_args)
+				)
+				self.assertIsNotNone(
+					multicast.__main__.McastDispatch().doStep("SAY", _fixture_SAY_args)
+				)
+				self.assertIsNotNone(
+					multicast.__main__.McastDispatch().doStep("SAY", _fixture_SAY_args)
+				)
+			except Exception:
+				p.join()
+				raise unittest.SkipTest(fail_fixture)
```

Comment:
Again, see issue #32

Reply:
That learning is not quite correct; the eventual introduction of the syntax of error chaining is in the scope of #32, so that changes to error handling is consolidated as it may involve dropping support for some older versions of python3 and needs careful designing.